Scouting/src/main/java/ca/team2706/scouting/mcmergemanager/steamworks2017/dataObjects/TeamNumber.java

```diff
@@ -0,0 +1,45 @@
+package ca.team2706.scouting.mcmergemanager.steamworks2017.dataObjects;
+
```

Comment:
Also not really sure what this class is doing, it looks like it's doing the same job as CommentList, but one comment at a time. Was this an earlier attempt that can be deleted?